release/xgboost_tests/workloads/train_small.py

```diff
@@ -9,14 +9,33 @@
 import json
 import os
 import time
+import yaml
 
 import ray
 from xgboost_ray import RayParams
 
 from ray.util.xgboost.release_test_util import train_ray
 
 if __name__ == "__main__":
-    ray.init(address="auto")
+    builder = ray.client("anyscale://train_small").cloud(
+        "anyscale_default_cloud")
+
+    dir = os.path.dirname(os.path.realpath(__file__))
+    compute_tpl_path = os.path.join(dir, "../tpl_cpu_small.yaml")
+    with open(compute_tpl_path, "r") as compute_tpl:
+        builder.cluster_compute(
+            yaml.load(
+                compute_tpl.read().replace("{{env[\"ANYSCALE_CLOUD_ID\"]}}",
+                                           builder._get_cloud_id("")),
+                Loader=yaml.FullLoader))
+
+    # This currently does not work on beta.anyscale.com, see
+    # https://github.com/anyscale/product/issues/4931
+    compute_env_path = os.path.join(dir, "../app_config.yaml")
+    with open(compute_env_path, "r") as cluster_env:
+        builder.cluster_env(yaml.load(cluster_env, Loader=yaml.FullLoader))
```

Comment:
alternatively, you can use the anyscale SDK to create a cluster env (look at the test_anyscale_development with connect) and reuse this cluster env in all the tests. This will significantly reduce the time it takes to run some of these tests.

Reply:
+1, we have the functionality in the e2e script already, and it will solve a couple of problems, e.g. rendering the yaml jinja2 template and also creating the test data (which is created in the docker image).

The name to the cluster env could then be passed via a command line argument

Reply:
Reusing the cluster created in the e2e script now.